.travis/deploy.sh

```diff
@@ -2,6 +2,18 @@ if [ ! -z "$TRAVIS_TAG" ]
 then
     echo "on a tag -> $TRAVIS_TAG and therefore we will do nothing. Tagged releases are deployed to sonatype manually."
 else
-    echo "not on a tag -> deploying to sonatype snapshot repo..."
+    echo "not on a tag -> deploying to dockerhub and sonatype..."
+    FCREPO_VERSION=$(./mvnw org.apache.maven.plugins:maven-help-plugin:3.1.0:evaluate -Dexpression=project.version -q -DforceStdout)
+    
+    echo "clone fcrepo4-labs/fcrepo-docker repository"
+    git clone https://github.com/fcrepo4-labs/fcrepo-docker.git
+    echo "building docker container"
+    cd fcrepo-docker
+    ./build.sh ../fcrepo-webapp/target/fcrepo-webapp-${FCREPO_VERSION}.war
```

Comment:
We might want to build the container within the default travis pipeline instead of in the deploy part and only include the push command here.

Reply:
Per our conversation in the meeting this morning,  we can defer until this change becomes necessary.